python/lsst/obs/decam/decamMapper.py

```diff
@@ -51,17 +51,33 @@ def __init__(self, inputPolicy=None, **kwargs):
 
         super(DecamMapper, self).__init__(policy, policyFile.getRepositoryPath(), **kwargs)
 
+        # Ensure each dataset type of interest knows all useful dataId keys
```

Comment:
I agree with @ktlim's comment on DM-5902 ("the need for this means that some unusual use is being made of dataId keys and that this is thus just a workaround for a more fundamental problem", etc).  I will drop this commit.